scripts/run-cool2multirescool.sh

```diff
@@ -2,5 +2,5 @@
 input=$1
 ncores=$2
 outprefix=$3
-cooler coarsegrain $input -n $ncores -o $outprefix.multires.cool
+cooler zoomify $input -n $ncores -o $outprefix.multires.cool
```

Comment:
Change to `cooler zoomify --balance $input -n $ncores -o $outprefix.multires.cool` to automatically balance all zoom levels after aggregating. Update cooler to 0.7.4 in `downloads.sh`.